src/lib/genesis_ledger_helper/genesis_ledger_helper.ml

```diff
@@ -168,9 +171,9 @@ module Ledger = struct
                   ] ;
               return None )
       | None ->
-          [%log warn]
+          [%log error]
```

Comment:
pls unrevert this log level + msg change- we don't want this to be an `error` until the rest of the logic is fixed to not try downloading when an explicit account list is provided.